components/Venue/venue.js

```diff
@@ -6,9 +6,10 @@ function Venue({ className, city }) {
 		<div>
 			<div style={{'--image-url': `url(${city.img})`}} 
 			className='relative w-[300px] h-[400px] sm:w-[250px] sm:h-[350px] card-bg bg-[image:var(--image-url)] flex items-center justify-center p-4 cursor-pointer'>
-				<div className='flex justify-between flex-col w-full h-full'>
+				 <div className="absolute  inset-0 bg-black opacity-30 rounded-3xl"/>
+				<div className='flex justify-between flex-col w-full h-full z-10'>
 					<div className='flex justify-between items-center'>
-						<div><div className={`border text-white text-md rounded-lg p-1 text-center mt-2 ${city.cfp ? 'block': 'hidden' }`}>cfp is open</div></div>
+						<div><div className={` text-md text-white text-md rounded-xl px-2 py-1 bg-[#3e4757]  text-sm text-center mt-2 ${city.cfp ? 'block': 'hidden' }`}>cfp is open</div></div>
```

Comment:
You are using multiple overriding text sizes. Also, try to make it pixel perfect including the `cfp is open` shadow and border radius as in [figma file ](https://www.figma.com/file/sS5ofLRtuFqDGFvlNBurns/AACoT'23%2F24?type=design&node-id=1457-1002&mode=design)

Reply:
@akkshitgupta  oh , i will fix that right away, and for the lucide-react, i dont know how it is install in that package, i will remove that .